src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -228,9 +235,11 @@ def _prepare_filelist(self):
 
     def run_conformance(self):
         gtest_parallel_path = os.path.join(self.__download_repo(GTEST_PARALLEL_URL, GTEST_PARALLEL_BRANCH), "thirdparty", "gtest-parallel", "gtest_parallel.py")
-        worker_num = os.cpu_count()
-        if worker_num > 2:
-            worker_num = worker_num - 1
+
+        if self._num_workers == -1:
```

Comment:
remove this logic, Use `cpu count -1` as default value

Reply:
Done